bavi/modules/random.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from random import choice
+
+log = logging.getLogger('bavi.modules.random')
+
+supported_split = [',', '|', '/', '\\']
+
+RESPONSE_MESSAGE = "Your choices: {0}, I chose: {1}."
+
+def init(bot):
+    bot.add_command('choose', choose_command)
+    bot.add_command('pick', choose_command)
+
+def choose_command(bot, source, target, message, **kwargs):
+    response = ''
+    result_list = make_list(message)
+    
+    if(len(result_list) > 1):
+        response = RESPONSE_MESSAGE.format(result_list, choice(result_list))
+    else:
+        response = "You need to give me more than one thing to choose!"
+        
+    bot.reply_to(
+                source,
+                target,
+                response
+        )
+        
+def make_list(message):
+    most_used_sep = []
+    result = []
+    
+    if(type(message) is str):
+
+        for i in supported_split:
+            most_used_sep.append(message.count(i))
+
+        seperator = supported_split[most_used_sep.index(max(most_used_sep))]
```

Comment:
Is there a reason we need to support this?  This kind of separator counting seems prone to accidentally choosing the wrong separator and behaving unexpectedly.

If we do want to keep multiple separators, I think it should be explicit somehow (like how `sed` allows you to do `s/foo/bar` or `s_foo_bar`; the first character after the `s` defines the separator).